examples/animation/animated_curve.rs

```diff
@@ -0,0 +1,265 @@
+//! Custom curve animation sampling
+use bevy::{animation::ActiveAnimation, input::common_conditions::input_just_pressed, prelude::*};
+
+fn main() {
+    App::new()
+        .add_plugins(DefaultPlugins)
+        .init_resource::<AnimationControl>()
+        .add_systems(Startup, setup)
+        .add_systems(
+            Update,
+            (
+                keyboard_control,
+                pickup_sprite.run_if(input_just_pressed(MouseButton::Left)),
+                drop_sprite.run_if(input_just_pressed(MouseButton::Right)),
+                update_text.run_if(resource_changed::<AnimationControl>),
+            ),
+        )
+        .add_systems(PostUpdate, myanimation_system)
+        .run();
+}
+
+///update text
```

Comment:
Doc comment should start with a capital letter, and has a space between it and `///`.